QuadRelate.Models/CPUPlayerFactory.cs

```diff
@@ -0,0 +1,23 @@
+﻿using QuadRelate.Contracts;
+using QuadRelate.Types;
+using System;
+
+namespace QuadRelate.Models
+{
+    public class CPUPlayerFactory : ICPUPlayerFactory
+    { 
+        public ICPUPlayer CreateCPUPlayer(string typeName)
+        {
+            ICPUPlayer CPUPlayer;
+
+            if (typeName == nameof(CPUPlayerRandom))
+            {
+                CPUPlayer = new CPUPlayerRandom();
```

Comment:
Could just use:
`return new CPUPlayerRandom();`
Unless you have a reason for the two lines.